src/features/user/MFV2/PublicProfileLayout/index.tsx

```diff
@@ -19,7 +20,7 @@ const PublicProfileLayout = ({ profile }: Props) => {
         )}
       </section>
       <section id="map-container" className={styles.mapContainer}>
-        Map
+        <ContributionsMap />
```

Comment:
It would be good if we could add a loader here in cases where map rendering or fetching of response from api takes time.

Reply:
There is a component called `<ProfileLoader/>`, you may try it. Make sure it fits the container perfectly though.